src/main/java/test/config/AppConfig.java

```diff
@@ -0,0 +1,41 @@
+package test.config;
+
+import java.util.Properties;
+import javax.sql.DataSource;
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+import test.model.User;
```

Comment:
```suggestion
import <>.model.User;
```
Let's add more useful package names. If you name package `test` in your test assignment - it will be minus.

Reply:
agree, renamed